src/components/Carousel/CarouselNav.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./CarouselNav.css";
+
+export default class CarouselNav extends React.Component {
+  prev = (wrapperRef) => {
+    const slide = wrapperRef.current;
+    slide.scrollLeft -= slide.offsetWidth;
+    if (slide.scrollLeft <= 0) {
+      slide.scrollLeft = slide.scrollWidth;
+    }
+  };
+
+  next = (wrapperRef) => {
+    const slide = wrapperRef.current;
+    slide.scrollLeft += slide.offsetWidth;
+    if (slide.scrollLeft >= slide.scrollWidth - slide.offsetWidth) {
+      slide.scrollLeft = 0;
+    }
+  };
```

Comment:
Estas funciones no deberían ser afectadas por la refactorización a functional components si utilizas el hook useRef donde creas la referencia.

Así que podemos conservarlas, solo hay que aplicar unos cambios para evitar repetir tanto la palabra `slide` para esto usaremos la famosa 'destructuración' de JavaScript. Quedaría así: 

```suggestion
  const prev = ({ current }) => {
    let { scrollLeft } = current;
    const { offsetWidth } = current;
    scrollLeft -= offsetWidth;
    if (scrollLeft <= 0) {
      scrollLeft = scrollWidth;
    }
  };

  const next = ({ current }) => {
    let { scrollLeft, scrollWidth, offsetWidth } = current;
    const { scrollWidth, offsetWidth } = current;
    scrollLeft += offsetWidth;
    if (scrollLeft >= scrollWidth - offsetWidth) {
      scrollLeft = 0;
    }
  };
```

Agrego la palabra `const` a las funciones para que podamos acceder a ellas con functional components, ya que si no recuerdo mal, la referencia de la palabra clave `this` se va a perder al ya no ser una clase.